inverter-eeprom-config/saved_EEPROM_configs/left_inv

```diff
@@ -57,19 +57,19 @@ Coast_Lo_EEPROM_(V)_x_100             ,      0
 Coast_Hi_EEPROM_(V)_x_100             ,      4  
 Accel_Max_EEPROM_(V)_x_100            ,    490  
 Pedal_Hi_EEPROM_(V)_x_100             ,    499  
-Motor_Torque_Limit_EEPROM_(Nm)_x_10   ,    500  
+Motor_Torque_Limit_EEPROM_(Nm)_x_10   ,   1320  
```

Comment:
Why do we have a torque limit set? Is this number at least much higher than we would request from the DCM?

Reply:
A torque limit of 132 Nm is set in the EEPROM based on the maximum current through the Emrax motor. The maximum torque request from the DCM is 90 Nm currently, so this is not a problem.